Tests/Functional/Hooks/ItemsProcFuncTest.php

```diff
@@ -47,7 +47,7 @@ public function canToollist()
             'items' => [
                 ['Score', 'tx_dlf_scoretool'],
                 ['Fulltext', 'tx_dlf_fulltexttool'],
-                ['', 'tx_dlf_adddocumenttool'],
+                ['', 'tx_dlf_multiviewaddsourcetool'],
```

Comment:
This should have a default label as well.

```suggestion
                ['Add Multiview Source', 'tx_dlf_multiviewaddsourcetool'],
```